os/arch/arm/src/amebasmart/amebasmart_i2c.c

```diff
@@ -301,8 +301,13 @@ static const struct amebasmart_i2c_config_s amebasmart_i2c0_config = {
 	//.busy_idle = CONFIG_I2C0_BUSYIDLE,
 	//.filtscl = CONFIG_I2C0_FILTSCL,
 	//.filtsda = CONFIG_I2C0_FILTSDA,
+#if CONFIG_RTL8730E_BOARD_REVISION >= 5
+	.scl_pin = PA_29,
+	.sda_pin = PA_28,
+#else
```

Comment:
https://github.com/Samsung/TizenRT/blob/9522f9b6f99fcf192de3cb2a6de3db240b46375f/os/arch/arm/src/amebasmart/amebasmart_i2c.c#L359-L365

I2C port 2 is already using PA_28 and PA_29 and it's the same mentioned in the data shared by h/w team. So, I am not sure if its the right change.

Reply:
This is a mistake on my side, please ignore this change, I have reverted it.